tsdb/engine/tsm1/data_file.go

```diff
@@ -185,6 +221,33 @@ func (d *directIndex) Entry(key string, t time.Time) *IndexEntry {
 	return nil
 }
 
+func (d *directIndex) Contains(key string) bool {
```

Comment:
Ah, OK -- I see it now. I quickly read `Entries` as a slice, but it's a function call.